services/ymax-planner/test/snapshots/deposit-tools.test.ts.md

```diff
@@ -30,22 +30,15 @@ Generated by [AVA](https://avajs.dev).
           brand: Object @Alleged: mock brand {},
           value: 475n,
         },
-        dest: 'USDNVault',
-        detail: {
-          usdnOut: 470n,
-        },
+        dest: 'USDN',
         src: '@noble',
       },
       {
         amount: {
           brand: Object @Alleged: mock brand {},
-          value: 305n,
+          value: 525n,
         },
         dest: '@Arbitrum',
-        fee: {
```

Comment:
Is the absence of these `fee` elements an issue?  Static fees are bogus so we need the automatic generation, but we need the static fees until they are computed.

Reply:
> Is the absence of these `fee` elements an issue?

yes; no Axelar stuff will work without it.

> ... so we need the automatic generation ...

See also

 - #11953